recipes/open62541/all/conanfile.py

```diff
@@ -101,6 +102,9 @@ def configure(self):
             del self.settings.compiler.cppstd
             del self.settings.compiler.libcxx
 
+        if self.options.subscription == "With Events":
+            self.output.warning("`{name}:subscription=With Events` is deprecated. Use `{name}:subscription=events` instead".format(name=self.name))  # Deprecated in 1.2.2
```

Comment:
I forgot to do this.
```suggestion
            self.output.warning("`{name}:subscription=With Events` is deprecated. Use `{name}:subscription=events` instead".format(name=self.name))  # Deprecated in 1.2.2
            self.options.subscription = "events"
```